File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/SatsMinsteytelse.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.kode.BorMedTypeCti;
import no.nav.domain.pensjon.regler.kode.MinsteytelseNivaCti;

import java.io.Serializable;

/**
 * @author Steinar Hjellvik (Decisive) - PK-11391
 */
public class SatsMinsteytelse implements Serializable {


    private double sats;

    private MinsteytelseNivaCti satsType;

    private BorMedTypeCti benyttetSivilstand;

    private boolean benyttetUngUfor;

    private boolean oppfyltUngUfor;

    /**
     * Angir om ung uføregaranti ikke er benyttet pga eksportforbud.
     */
    private boolean eksportForbudUngUfor;

}

File: src/main/java/no/nav/pensjon/regler/domain/beregning2011/SatsMinsteytelse.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.kode.BorMedTypeCti
import no.nav.pensjon.regler.domain.kode.MinsteytelseNivaCti
import java.io.Serializable

class SatsMinsteytelse : Serializable {
    var sats = 0.0
    var satsType: MinsteytelseNivaCti? = null
    var benyttetSivilstand: BorMedTypeCti? = null
    var isBenyttetUngUfor = false
    var isOppfyltUngUfor = false

    /**
     * Angir om ung Uføregaranti ikke er benyttet pga eksportforbud.
     */
    var isEksportForbudUngUfor = false

}